code/src/components/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+const Form = ({ handleFormSubmit }) => {
+  const [message, setMessage] = useState("");
+  const [alert, setAlert] = useState(false);
+  const onMessageChange = (e) => {
+    const count = message.length;
+    if (count > 140) {
+      setAlert(true);
+    } else {
+      setAlert(false);
```

Comment:
I thought this was going to display an alert if I tried to send a message with more than 140 characters. But is this only for the message not to send when it is more than 140?